packages/accordion/src/accordion.tsx

```diff
@@ -58,6 +58,10 @@ export const Accordion = forwardRef<AccordionProps, "div">(
       [context, reduceMotion],
     )
 
+    const rootStyles: SystemStyleObject = {
```

Comment:
Could this be inlined to the __css prop?

Reply:
sure!

Reply:
Awesome 😊

Reply:
@TimKolberger adjusted it